src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
A cleaner approach would be to extract all properties from the person here, to avoid typing `person.` each time.